res/controllers/Hercules DJControl Compact.midi.xml

```diff
@@ -12,6 +14,28 @@
         </scriptfiles>
         <controls>
             <control>
+                <group>[EqualizerRack1_[Channel1]_Effect1]</group>
+                <!-- Set the key on the next line to "parameter3" to make the
+                mid knob for Deck A adjust the high eq instead. -->
+                <key>parameter2</key>
+                <status>0xB0</status>
+                <midino>0x3B</midino>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
+                <group>[EqualizerRack1_[Channel2]_Effect1]</group>
+                <!-- Set the key on the next line to "parameter3" to make the
```

Comment:
Would it be better to use the <description> element for this? That shows up in the mapping GUI.